src/main/java/at/aau/ase/cl/mapper/OpenLibraryMapper.java

```diff
@@ -0,0 +1,141 @@
+package at.aau.ase.cl.mapper;
+
+import at.aau.ase.cl.api.model.BookFormat;
+import at.aau.ase.cl.client.openlibrary.model.Author;
+import at.aau.ase.cl.client.openlibrary.model.Book;
+import at.aau.ase.cl.client.openlibrary.model.KeyValue;
+import at.aau.ase.cl.domain.AuthorEntity;
+import at.aau.ase.cl.domain.BookEntity;
+import at.aau.ase.cl.domain.Languages;
+import io.quarkus.logging.Log;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+import org.mapstruct.factory.Mappers;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+@Mapper
+public interface OpenLibraryMapper {
+    OpenLibraryMapper INSTANCE = Mappers.getMapper(OpenLibraryMapper.class);
+
+    Map<BookFormat, Set<String>> ALTERNATIVE_FORMATS = Map.of(
```

Comment:
I think the alternative formats given here are sufficient but we might have to add further formats later on. Please add the BookFormat class and this information about the mapping to [our domain model](https://github.com/AAU-ASE24W-group-E/crowd-library/blame/main/docs/domain-analysis/domain-diagram.md)

Reply:
thanks for hint! -> diagram updated